recipe/meta.yaml

```
@@ -10,24 +10,22 @@ source:
   sha256: dab68723994c6b4503f20f543dad80a1a2177791520115313a80a6d1167e69a4
 
 build:
-  noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<36]
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - datasets >=1.8.0
+    - python
     - gdown >=4.0.0
-    - numpy >=1.16.2
-    - pandas
-    - python >=3.6
-    - pytorch >=1.2.0
+    - numpy
```

```suggestion
    - numpy >=1.16.2
```
We need to keep the lower bound to make sure that users are not surprised if they try to install the package using an older numpy.